### PR TITLE
Fix sanitization issues within quotes

### PR DESCRIPTION
**Pull request by Owen1212055**
[contributing]: https://github.com/DV8FromTheWorld/JDA/wiki/5%29-Contributing

## Pull Request Etiquette

- [X] I have checked the PRs for upcoming features/bug fixes.
- [X] I have read the [contributing guidelines][contributing].

### Changes

- [X] Internal code
- [ ] Library interface (affecting end-user code) 
- [ ] Documentation
- [ ] Other: \_____ 

Closes Issue: #1360 

## Description

This fixes the issue causing characters not to sanitize when put into multiple multiline quotes.

Basically, instead of breaking the quotes by their ending we break them up at the start.

### CODE REVIEW

**Comment by Owen1212055** [comment]
I apologize for that mess, I forgot to test some of the other methods as well and then ran into other problems!
